webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx

```diff
@@ -320,6 +320,10 @@ const TableActionBar: React.FC<Props> = ({
       key: action,
       label: action,
     }));
+    const divider: MenuDivider = { type: 'divider' };
+    menu.splice(1, 0, divider);
+    menu.splice(6, 0, divider);
```

Comment:
nit: splicing them in like this at fixed indexes seems like it would be easy to forget to change if we change the batchActions. Maybe you could break batchActions into multiple arrays and join them with dividers, or use indexOf to find the location you want to splice them into programmatically?

Reply:
great suggestion

Reply:
I put in the comments the actual listing so it will be more obvious what the menu item looks like.
Also created grouping as you suggested instead of injecting dividers into the list (great idea).